pkg/controller/acmeorders/sync_test.go

```diff
@@ -190,7 +190,7 @@ dGVzdA==
 					testpkg.NewAction(coretesting.NewCreateAction(cmacme.SchemeGroupVersion.WithResource("challenges"), testAuthorizationChallenge.Namespace, testAuthorizationChallenge)),
 				},
 				ExpectedEvents: []string{
-					`Normal Created Created Challenge resource "testorder-1335133199" for domain "test.com"`,
+					`Normal Created Created Challenge resource "testorder-2381786951" for domain "test.com"`,
```

Comment:
Where does this suffix come from? Why has it changed?

Reply:
I just assumed that this was some automatically generated hash of the object that changed due to the added `FailurePolicy` property.

Bad assumption probably, I was going to ask about this, but haven't gotten around to it yet. :)

Reply:
This is now reverted.